.circleci/config.yml

```diff
@@ -0,0 +1,146 @@
+
+version: 2
+jobs:
+  build:
+    docker:
+      # openjdk provides Scala dependencides. We may want to consider rolling
+      # our own image to also include JavaScript dependencies. Would speed up CI
+      # at the cost of maintaining a Docker image.
+      - image: openjdk:8
+        environment:
+          CI_TERRIBLENESS: 30.seconds
+        # Run two containers, $CIRCLE_NODE_TOTAL = 2, $CIRCLE_NODE_INDEX = [0|1]
+        parallelism: 2
+    working_directory: ~/linkerd
+    # All run steps execute in both containers unless otherwise specified.
+    # Deploy steps execute on container 0.
+    steps:
+      # Check out linkerd source code into ~/linkerd.
+      - checkout
+
+      # This allows us to run docker commands Since we're already in a docker
+      # container, this step sets up a remote docker environment for us.
+      - setup_remote_docker
+
+      # cat the entire contents of the project/ directory, to be used in cache
+      # keys.
+      - run:
+          name: Generate cache seeds
+          command: |
+            set -x
+            find ~/linkerd/project/ -type f -exec cat {} \; > project-seed.txt
+
+      # Restore build dependencies and downloaded files from the save_cache
+      # step. We use project-seed.txt, generated from the entire contents of the
+      # project/ directory, as a proxy for Scala dependency changes. We use
+      # package.json as a proxy for JavaScript dependency changes.
+      - restore_cache:
+          key: linkerd-{{ checksum "project-seed.txt" }}-{{ checksum "admin/src/main/resources/io/buoyant/admin/package.json" }}
+
+      # Install Docker client to enable running commands on the remote Docker,
+      # set up above.
+      - run:
+          name: Install Docker client
+          command: |
+            set -x
+            if [ ! -f docker-17.03.0-ce.tgz ]; then
+              curl -sLO https://get.docker.com/builds/Linux/x86_64/docker-17.03.0-ce.tgz
+            fi
+            tar -xz -C /tmp -f docker-17.03.0-ce.tgz
+            mv /tmp/docker/* /usr/bin
+
+      # Download and install Node.js. Skip download if already in cache.
+      - run:
+          name: Install Node.js
+          command: |
+            set -x
+            if [ ! -f node-v6.10.2-linux-x64.tar.xz ]; then
+              curl -sLO https://nodejs.org/dist/v6.10.2/node-v6.10.2-linux-x64.tar.xz
+            fi
+            tar -xf node-v6.10.2-linux-x64.tar.xz
+            cp -a node-v6.10.2-linux-x64/* /usr/
+
+      # Download and install protoc. Skip download if already in cache.
+      - run:
+          name: Install protoc
+          command: |
+            set -x
+            if [ ! -f protoc-3.1.0-linux-x86_64.zip ]; then
+              curl -sLO https://github.com/google/protobuf/releases/download/v3.1.0/protoc-3.1.0-linux-x86_64.zip
+            fi
+            unzip -p protoc-3.1.0-linux-x86_64.zip bin/protoc > /bin/protoc
+            chmod 755 /bin/protoc
```

Comment:
i don't think this is relevant any longer since we now use `./protoc` (which is what's actually called from within sbt).  And, instead, we have to cache `.protoc`